src/Components/TaskList.js

```diff
@@ -0,0 +1,14 @@
+
+import React from 'react';
+
+import Task from './Task';
+
+const TaskList = ({ tasks, setTaskList, taskToBeEdited, setTaskToBeEdited, newDescription, setNewDescription }) => {
+    return (
+        <div className="tasks-container">
+            {tasks.map(task => <Task description={task.description} key={task.id} tasks={tasks} setTaskList={setTaskList} id={task.id} taskToBeEdited={taskToBeEdited} setTaskToBeEdited={setTaskToBeEdited} newDescription={newDescription} setNewDescription={setNewDescription}/>)}
```

Comment:
When using `map` first check if the variable being referred is defined or not. If not defined return null. Preferrably use a `ternary` operator in this case.